src/storage_manifests.py

```diff
@@ -142,3 +143,24 @@ def evaluate(self) -> Optional[str]:
             if not self.config.get(prop):
                 return f"Storage manifests waiting for definition of {prop}"
         return None
+
+class UpdateControllerPlugin(Patch):
+    """Update the controller args in Deployments."""
+
+    def __call__(self, obj):
+        """Update the controller args in Deployments."""
+        if not any(
+            [
+                (obj.kind == "DaemonSet" and obj.metadata.name == "csi-cinder-nodeplugin"),
+                (obj.kind == "Deployment" and obj.metadata.name == "csi-cinder-controllerplugin"),
+            ]
+        ):
+            return
```

Comment:
My understand was this only affected the "csi-cinder-controllerplugin"

```suggestion
        if not (obj.kind == "Deployment" and obj.metadata.name == "csi-cinder-controllerplugin"):
            return
```

Reply:
thanks!